docs/assessment.md

```diff
@@ -308,31 +307,27 @@ Information about the group configuration can be set in the `infoAssessment.json
 | `studentGroupJoin`   | boolean | false   | Allow students to join other groups by join code.  |
 | `studentGroupLeave`  | boolean | false   | Allow students to leave groups.                    |
 
-Please notice: changing an assessment from group -> individual or vice versa after students have started working on it will cause student work to be lost.
+Note that changing an assessment from individual to group or vice versa after students have started working on it will cause student work to be lost.
 
-### Instructor options for groupWork
+### Managing groups
 
-![Instructor group assignment page](groupwork_instructor_interface.png)
+Instructors can manage groups on the "Groups" tab of an assessment.
 
-Underneath the "Groups" tab in an assessment, instructors have three ways of assigning students to different groups:
+To create specific groups, upload a CSV file in the following format:
```

Comment:
```suggestion
To create multiple groups in batch, upload a CSV file in the following format:
```

Reply:
Tweaked your comment a bit: "multiple groups in batch" -> "groups in bulk", sound good?

Reply:
Sure, that's fine. I just wanted to avoid confusion with individual operations that don't require upload.